tests/Unit/PointwiseFunctions/AnalyticSolutions/GrMhd/Test_BondiMichel.cpp

```diff
@@ -111,6 +121,20 @@ void test_variables(const DataType& used_for_size) {
                       mag_field_strength),
       used_for_size);
 }
+
+void test_solution() noexcept {
+  grmhd::Solutions::BondiMichel solution(1.0, 50.0, 1.3, 1.5, 0.24);
```

Comment:
I assume that I will have to change this away from 1.0 once I've fixed BondiMichel to test that BondiMichel is a solution even for M!=1.0

Reply:
yes, this test would currently fail if you change the mass